tsconfig.json

```diff
@@ -13,9 +13,8 @@
       "dom",
       "es6"
     ],
-    "types": [
-      "jasmine",
-      "node"
+    "typeRoots": [
```

Comment:
Removing this change from this branch as it doesn't seem to be throwing the same error anymore. I guess I fixed it inadvertently.